pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -11,8 +11,10 @@
 from pandas.compat.numpy import np_datetime64_compat
 
 from pandas.core.series import Series
-from pandas.tseries.frequencies import (_offset_map, get_freq_code,
-                                        _get_freq_str, _INVALID_FREQ_ERROR,
+
+from pandas._libs.tslibs.frequencies import (get_freq_code, get_freq_str,
+                                             _INVALID_FREQ_ERROR)
+from pandas.tseries.frequencies import (_offset_map,
```

Comment:
_offset_map? future PR?

Reply:
I think _offset_map is intended to be private, don't think its imported outside of tests.  (Also if we ever make the immutable thing work, we'll be able to get rid of offset_map altogether)